tests/integration/cassettes/TestIdentityProvidersResource.test_list_idps.yaml

```diff
@@ -0,0 +1,223 @@
+interactions:
+- request:
```

Comment:
Curious, why do you need these YAML files? Are you mocking the server responses instead of hitting the actual API?

Reply:
Yes, It is to be able to mock the request/response to bypass the end to end, however, you can still run e2e and bypass the casettes

Reply:
Exactly, just to make the tests more efficient and reliable. The YAMLs contain actual runs against the API

Reply:
I see, why did you guys decide to follow that approach? I'm just curious to know the background since other SDKs (expect authn.SDK) hit the Okta API.

Reply:
Overall, it was done for performance. The tests shave off a lot of time with these cassettes and reduce our reliance on the backend services

Reply:
I see. Thanks for clarifying.